### PR TITLE
OPS-935: Add enhanced networking to cloud migrations

### PR DESCRIPTION
**Pull request by maxrothman**

### CODE REVIEW

**Comment by fredsmith** [comment]
:+1:

File: playbooks/edx-east/edx_service.yml [hunk 1]
```diff
@@ -317,4 +318,35 @@
...
         opts: "{{ item.options | default('defaults') }}"
-      with_items: volumes
\ No newline at end of file
+      with_items: volumes
+
+
+#Currently only supported in non-asg mode, when auto_scaling_service==false
+#<http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enable-enhanced-networking>
+#Done with local actions to avoid blocking on iteration
+- name: Enable enhanced networking
```
Comment (feanil) [inline playbooks/edx-east/edx_service.yml:327]: No need to have a separate play for this.

**Comment by feanil** [comment]
:+1:  once the one comment is resolved.

**Comment by maxrothman** [comment]
@feanil agreed with merging without merging the plays to avoid starting another testing cycle

**Pull request merged**